packages/zendesk/scripts/build.mjs

```diff
@@ -3,7 +3,6 @@ import fs from "fs-extra";
 const currentDir = process.cwd();
 const destinationDir = process.env.INIT_CWD + "/dist";
 const env = process.env.ENV;
-const apps = process.env.APPS && process.env.APPS.split(" ");
 const COPY_LIST = ["assets", "translations", "manifest.json"];
```

Comment:
Can you please add `"zcli.apps.config.json"` to the list?

Reply:
added :)
can you retest with your config to see if this works as intended?